Lib/xml/etree/ElementTree.py

```diff
@@ -679,6 +679,7 @@ def iterfind(self, path, namespaces=None):
     def write(self, file_or_filename,
```

Comment:
This should add a unit test to test it's behavior. Verify it whether the requirements are met.

Reply:
Added some unit tests but there is'nt an issu to add an issue title.

and i think the change has little impact on Python users.